### DISCUSSION TITLE
Ubuntu Failed to start D-Bus system message bus error

### DISCUSSION THREAD

**Discussion by Princy-Malhotra** [actions/virtual-environments#7773] [General questions]
I am getting this error when I try to boot Ubuntu: 
 
Failed to start System Logging Service. 
Failed to start D-Bus System Message Bus. 
Failed to start RPC Portman Service. 
Failed to start User Login Management.

How to resolve this? I am using Ubuntu 22.04. Any help is welcome.

**Comment by ilia-shipitsin** [comment]
do you have a link to failed build ?

**Reply by Princy-Malhotra** [reply]
> do you have a link to failed build ?

Actually, I am running it on my local system with Windows 11 dual boot.

**Reply by ilia-shipitsin** [reply]
can you provide more details, please ?
